README.md

```diff
@@ -1,96 +1,155 @@
 
 ## Contributing
 
-We appreciate feedback and contribution to this repo! Before you get started, please see the following:
+We appreciate your feedback and contributions to the project! Before you get started, please review the following:
 
 - [Auth0's general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
 - [Auth0's code of conduct guidelines](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
-- [This repo's contribution guide](CONTRIBUTING.md)
+- [The Auth0 PHP SDK contribution guide](CONTRIBUTING.md)
 
 ## Support + Feedback
 
-- Use [Community](https://community.auth0.com/) for usage, questions, specific cases
-- Use [Issues](https://github.com/auth0/auth0-PHP/issues) here for code-level support and bug reports
-- Customers with a paid Auth0 subscription can use the [Support Center](https://support.auth0.com/) to submit a ticket to our support specialists.
+- The [Auth0 Community](https://community.auth0.com/) is a valuable resource for asking questions and finding answers, staffed by the Auth0 team and a community of enthusiastic developers
+- For code-level support (such as feature requests and bug reports) we encourage you to [open issues](https://github.com/auth0/auth0-PHP/issues) here on our repo
+- For customers on [paid plans](https://auth0.com/pricing/), our [support center](https://support.auth0.com/) is available for opening tickets with our knowledgeable support specialists
+
+Further details about our support solutions are [available on our website.](https://auth0.com/docs/support)
 
 ## Vulnerability Reporting
 
 Please do not report security vulnerabilities on the public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
 
 ## What is Auth0?
 
-Auth0 helps you to easily:
+Auth0 helps you to:
 
-- implement authentication with multiple identity providers, including social (e.g., Google, Facebook, Microsoft, LinkedIn, GitHub, Twitter, etc), or enterprise (e.g., Windows Azure AD, Google Apps, Active Directory, ADFS, SAML, etc.)
-- log in users with username/password databases, passwordless, or multi-factor authentication
-- link multiple user accounts together
-- generate signed JSON Web Tokens to authorize your API calls and flow the user identity securely
-- access demographics and analytics detailing how, when, and where users are logging in
-- enrich user profiles from other data sources using customizable JavaScript rules
+- Add authentication with [multiple authentication sources](https://docs.auth0.com/identityproviders), either social like **Google, Facebook, Microsoft Account, LinkedIn, GitHub, Twitter, Box, Salesforce, amont others**, or enterprise identity systems like **Windows Azure AD, Google Apps, Active Directory, ADFS or any SAML Identity Provider**.
```

Comment:
- amont => among. 
- Not sure why `Account` got added to `Microsoft`, all mentioned social providers require `Accounts` with the specific social providers.
- I am not entirely sure, but to me `among others` sounds weird in the way this paragraph is phrased,

Reply:
I'll tweak the wording a bit on this, see what you think